pandas/core/reshape/reshape.py

```diff
@@ -923,11 +923,17 @@ def get_empty_Frame(data, sparse):
 
     number_of_cols = len(levels)
 
+    py2_prefix_sep_is_unicode = isinstance(prefix_sep, text_type)
     if prefix is not None:
-        dummy_strs = [u'{prefix}{sep}{level}' if isinstance(v, text_type)
-                      else '{prefix}{sep}{level}' for v in levels]
-        dummy_cols = [dummy_str.format(prefix=prefix, sep=prefix_sep, level=v)
-                      for dummy_str, v in zip(dummy_strs, levels)]
+        py2_prefix_is_unicode = isinstance(prefix, text_type)
```

Comment:
can you make a little helper function here so that you can do this as a list-comprehension